disco_aws_automation/disco_elastigroup.py

```diff
@@ -0,0 +1,242 @@
+"""Contains DiscoElastigroup class that orchestrates AWS Spotinst Elastigruops"""
```

Comment:
Elstigroup (without **c**) is a Spotinst terminology and I wanted to preserve that.